### ISSUE TITLE
Add function in directory-indexer.py to add existing build

### ISSUE THREAD

**Issue by justaddcoffee** [Knowledge-Graph-Hub/kg-covid-19#359]
(Note: this template is for requesting new features in the software, not new datasets to ingest. Use the "Dataset request" for that.)

### Describe the desired behavior

Right now in Jenkins we manually add a (blank) directory for each existing build in our projectdir:
https://github.com/Knowledge-Graph-Hub/kg-covid-19/blob/4a1fda723602b90f2ef22da43452f15a82e883ed/Jenkinsfile#L175
on the command-line.

A better solution would be to add this functionality to directory-indexer.py on [our fork of go-site](https://github.com/justaddcoffee/go-site)

## Additional context

Basically we'd just add flags for the bucket and project directory, and some ignore args to ignore directories like `raw` and `current`. The existing builds would be added to the HTML as href links.
